src/form/select.rs

```diff
@@ -155,19 +155,21 @@ impl Component for MultiSelect {
     fn view(&self) -> Html {
         let mut classes = Classes::from("select is-multiple");
         if let Some(extra) = &self.props.classes {
-            classes = classes.extend(extra);
+            classes.push(extra);
         }
         if let Some(size) = &self.props.size {
             classes.push(&size.to_string());
         }
         if self.props.loading {
             classes.push("is-loading");
         }
+
+        let size: Option<String> = self.props.size.as_ref().map(|size| size.to_string());
         html! {
             <div class=classes>
                 <select
                     multiple=true
-                    size=self.props.list_size
+                    size=size
```

Comment:
Looks like this should still reference `list_size`, not `size`. Size is interpolated into the classes, `list_size` is used to control the size of the multi-select list.

Reply:
Yes, that's correct. Roger that, done in https://github.com/thedodd/ybc/pull/20/commits/bcd6bf395f085ff3641415fd69ca88be8a422041.